machine_management/control_plane_machine_management/cpmso_provider_configurations/cpmso-config-options-aws.adoc

```diff
@@ -1,58 +1,27 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="cpmso-config-options-aws"]
-= Control plane configuration options for Amazon Web Services
 include::_attributes/common-attributes.adoc[]
+[id="cpmso-config-options-aws"]
+= Changing control plane configuration options for {aws-full}
 :context: cpmso-config-options-aws
 
 toc::[]
 
-You can change the configuration of your {aws-first} control plane machines and enable features by updating values in the control plane machine set.
-When you save an update to the control plane machine set, the Control Plane Machine Set Operator updates the control plane machines according to your configured xref:../../../machine_management/control_plane_machine_management/cpmso-managing-machines.adoc#cpmso-feat-config-update_cpmso-managing-machines[update strategy].
+[role="_abstract"]
+You can change the configuration of your {aws-first} control plane machines by updating values in the control plane machine set.
+You might make updates to reflect changes to your infrastructure or environment.
```

Comment:
I get the motivation of this second sentence (very directly explaining the "why" aspect of this assembly), but despite being grammatically correct, I can't help but think that the average reader might find the phrasing almost overly formal or stiff. Perhaps some more direct call to action such as this might work as well?
```suggestion
Perform these updates to reflect changes in your infrastructure or environment.
```
I can see how this phrasing might make it sound more mandatory than it really is, but I think the "can" in the first sentence does enough to convey things as optional.

Either way feel free to keep as-is, this is just a thought.